docs/installation.md

```diff
@@ -48,56 +49,56 @@ For Ubuntu or another Debian-based Linux distribution, run these commands:
 
 Release archives also are available from <https://github.com/Azure/AvereSDK/releases>. (This is the same release available with pip but in a standalone archive format.)
 
 You can test that the script is active by issuing the help command.
 
     vfxt.py --help
 
-If you see the help text, the test was successful. If you do not see the vfxt.py help text, check the software requirements and try the installation again.
-
-## Configuring the Cloud Environment
+If you see the help text, the test was successful. If you do not see the vfxt.py help text, check the software requirements and try the installation again. 
 
-Before you can use vfxt.py to create or modify Avere clusters, you must have a cloud account set up with a supported provider, and appropriate environment configuration.
+## Configuring the Cloud Environment 
 
-Regardless of whether you use vfxt.py on a virtual machine inside your cloud environment, or from a console outside the cloud, the following basic requirements apply:
+Before you can use vfxt.py to create or modify Avere clusters, you must have a cloud account set up with a supported provider, and appropriate environment configuration. 
 
-* vfxt.py must be able to make contact with the cloud provider's API endpoints. A proxy service might be required.
-* vfxt.py must be able to authenticate to the cloud environment.
-* vfxt.py must have permission to create and destroy virtual machine instances in the cloud environment.
+Regardless of whether you use vfxt.py on a virtual machine inside your cloud environment, or from a console outside the cloud, the following basic requirements apply: 
 
-The details vary greatly by cloud provider. Read the detailed Avere vFXT Installation Guide for complete information about setting up projects, network configuration, security and privileges, and other vital information before attempting to use vfxt.py.
+* vfxt.py must be able to make contact with the cloud provider's API endpoints (a proxy service might be required). 
+* vfxt.py must be able to authenticate to the cloud environment. 
+* vfxt.py must have permission to create and destroy virtual machine instances in the cloud environment. 
 
-* vFXT Installation Guide for Microsoft Azure (vfxt.py version in development, read current documentation [here](<https://docs.microsoft.com/azure/avere-vfxt/avere-vfxt-deploy-plan>))
+The details vary greatly by cloud provider. Read the detailed Avere vFXT Installation Guide for your cloud compute provider for complete information about setting up projects, network configuration, security and privileges, and other vital information before attempting to use vfxt.py. 
 
-These sections of this documentation also give provider-specific information about using vfxt.py on particular cloud systems.
+* vFXT Installation Guide for Microsoft Azure (coming soon, read current documentation [here](<https://github.com/Azure/Avere/blob/master/README.md>))
```

Comment:
Hmm I don't think anything here is coming soon. Should this link to https://docs.microsoft.com/en-us/azure/avere-vfxt/ ?